roles/network_plugin/calico/tasks/install.yml

```diff
@@ -28,6 +28,7 @@
     src: "{{ etcd_cert_dir }}/{{ item.s }}"
     dest: "{{ calico_cert_dir }}/{{ item.d }}"
     state: hard
+    mode: 0750
```

Comment:
0640 ?

Reply:
Indeed 0640 should be enough, +x for crt/pem files is a bit weird 😛

Reply:
That's right. I guess I was sleepwalking😪